console.js

```diff
@@ -16,7 +17,7 @@ try {
     console.log('Connected to default', endpoint);
   }
 
-  var context = repl.start(endpoint +'> ').context;
+  var context = repl.start('> ').context;
```

Comment:
instead of just using `>` maybe we should use `myURL.host + '> '` since we already have `url` package here.
more info on `url` in node https://nodejs.org/api/url.html#url_url_host

Reply:
** e.g. `myURL.host + '> '`  should change it to `api.nodesmith.io> `